frontend/pages/admin/IntegrationsPage/cards/ConditionalAccess/ConditionalAccess.tsx

```diff
@@ -63,14 +80,16 @@ const DeleteConditionalAccessModal = ({
           <Button
             type="button"
             variant="alert"
-            onClick={onDelete}
-            isLoading={isUpdating}
+            onClick={handleDelete}
+            isLoading={isDeleting}
+            disabled={isDeleting}
```

Comment:
Why have this new state instead of relying on the parent `isUpdating`? Could you have these
`disbled` props depend on `isUpdating` to achieve the same result without this new `useState`?

Reply:
I tried but that didn't work. So I ended up emulating what other Delete modals that have Delete buttons do.